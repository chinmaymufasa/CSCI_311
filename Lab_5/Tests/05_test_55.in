2
168
4
2
192
5
2
186
4
2
106
4
2
76
4
-1
