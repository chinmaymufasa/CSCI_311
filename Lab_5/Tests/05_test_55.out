168 
168 192 
186 168 192 
186 168 106 192 
186 106 76 168 192 
